src/pages/Main/Main.js

```diff
@@ -1,12 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import './Main.scss';
+import Header from './components/Header/Header';
+import Section from './components/Section/Section';
+
+const Main = () => {
+  const [headerData, setHeaderData] = useState([]);
+  const [sectionData, setSectionData] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/headerData.json', {
```

Comment:
```suggestion
    fetch('/data/headerData.json', {
```
http://localhost:3000 은 생략 가능합니다. 포트 번호가 달라지면(ex. localhost:3001) 코드 전체를 바꿔야 하기 때문에 수정해주시기 바랍니다.

Reply:
수정완료 감사합니다.